chapter_preliminaries/probability_vn.md

```diff
@@ -124,7 +132,14 @@ The output of the `np.random.multinomial` function is another vector of the same
 its value at index $i$ is the number of times the sampling outcome corresponds to $i$.
 -->
 
-*dịch đoạn phía trên*
+Tiếp theo, ta sẽ cần tung xúc xắc.
+Trong ngành thống kê, ta gọi quá trình thu các mẫu từ phân phối xác suất là quá trình *lấy mẫu*.
+Phân phối mà gán các xác suất cho các lựa chọn rời rạc (*discrete choices*) được gọi là *phân phối đa thức* (*multinomial distribution*).
```

Comment:
```suggestion
Phân phối mà gán các xác suất cho các lựa chọn rời rạc (discrete choices) được gọi là *phân phối đa thức* (*multinomial distribution*).
```

Reply:
Mình nhớ đợt anh @tiepvupsu có đưa lên quy ước chú thích trên slack. Mình lưu ở google keep như này.

tiếng Việt (*English*)
hoặc

tiếng Việt (_English_)
Một dấu (*) hay (_) đều có nghĩa là in nghiêng.

Nếu từ gốc trong bản tiếng Anh có in nghiêng hay in đậm thì giữ nguyên:

*tiếng Việt in nghiêng* (*English*)
hoặc

**tiếng Việt in đậm** (*English*)
Nếu tiếng Anh có viết tắt thì cùng thêm trong ngoặc. Từ viết tắt không in nghiêng vì mình không dịch từ viết tắt đó.

phân tích thành phần chính (*principle component analysis* -- PCA)